### DISCUSSION TITLE
Docking station, DisplayLink, Black Screens on an M2 Mac

### DISCUSSION THREAD

**Discussion by IStartededWithAPL** [f1multiviewer/issue-tracker#399] [General]
Hi All, not sure if this is the right place to put this, I did look around. 

I have an MacBook M2 with a Dell 600 docking station using three monitors with DisplayLink.

Problem:
When I have DisplayLink loaded, then most of the time I get a black screen. I can hear the video, but no video. I say most of the time because it won't work for newish videos, but it works for the playbacks when they are older.

Observation:

- If I have a video window playing on the MacBook monitor and I unplug the docking station, the video appears and then goes away went I plug in back in.
- Somebody on the web says that DisplayLink uses the "screen record" API so it will never work with DRM (It sounds plausible, but not sure if it's correct)

Solution:
I quit DisplayLink, loose one of the two monitors and live with a two monitor setup vs a 3 monitor setup

**Comment by f1multiviewer** [comment]
MultiViewer only supports HDCP-compatible display output. A lot of docking stations, adapters, splitters and things like DisplayLink don't support HDCP. There's not really anything we can do to solve this.
